modules/generator/processor/spanmetrics/spanmetrics.go

```diff
@@ -108,7 +128,7 @@ func (p *Processor) aggregateMetricsForSpan(svcName string, rs *v1.Resource, spa
 	p.spanMetricsDurationSeconds.ObserveWithExemplar(registryLabelValues, latencySeconds, tempo_util.TraceIDToHexString(span.TraceId))
 }
 
-func sanitizeLabelNameWithCollisions(name string) string {
+func sanitizeLabelNameWithCollisions(name string, intrinsicDimensions []string) string {
```

Comment:
sanitizing attribute names only if they conflict with _configured_ intrinsics has some weird properties. if someone configures `status_message` as a normal attribute and then later adds the matching intrinsic then the name of the normal attribute label changes. this may break dashboards or queries b/c it changes the meaning of their metric labels. similar situations can occur when disabling intrinsic dimensions.

my instinct is to sanitize against all intrinsics even if they are not configured.

Reply:
Done